mico-core/src/test/java/io/github/ust/mico/core/ServiceResourceTests.java

```diff
@@ -55,10 +55,12 @@
 import java.util.*;
 
 import static io.github.ust.mico.core.ApplicationResourceTests.INTERFACES_LIST_PATH;
+import static io.github.ust.mico.core.GitHubCrawlerIntegrationTests.HELLO_REPO_SUB_DIR_DOCKERFILE;
```

Comment:
These new imports are unused → Discard all changes made in `ServiceResourceTests`

Reply:
Done